src/app/app.module.ts

```diff
@@ -1,18 +1,26 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { PostsModule } from './posts/posts.module';
+import { HttpClientModule } from '@angular/common/http';
+import {HomeModule} from './home/home.module';
```

Comment:
Always add space between the brackets: { HomeModule }

Reply:
Ok. Done